File: trunk/Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/OrderManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IOrderManager
	'	Inherits IManagerBase(Of Order, System.Int32)
	'End Interface

	Partial Class OrderManager
		Inherits ManagerBase(Of Order, System.Int32)
		Implements IOrderManager
	End Class
End Namespace

File: Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/OrderManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IOrderManager : IManagerBase<Order, int>
    {
	}
	
	partial class OrderManager : ManagerBase<Order, int>, IOrderManager
    {
	}
}